sdk/parents/azure-data-sdk-parent/pom.xml

```diff
@@ -228,6 +228,9 @@
             <footer>Visit the &lt;a href="https://docs.microsoft.com/java/azure/"&gt;Azure for Java Developers&lt;/a&gt;site for more Java documentation, including quick starts, tutorials, and code samples.</footer>
             <linksource>false</linksource>
             <excludePackageNames>com.microsoft.azure.template:*.impl.*:*.implementation.*</excludePackageNames>
+            <sourceFileExcludes>
```

Comment:
This is the parent pom for Track 1 libraries. There should be no changes to this.

Reply:
Oh that's still from when we used the wrong parent. Will revert.